packages/backend/src/storage/storage.test.ts

```diff
@@ -310,12 +310,44 @@ describe('Files', () => {
     const fileContent: FileContent = {
       path: path.join(__dirname, '/testUtils/test-image.png'),
       name: 'test-image',
-      ext: 'png'
+      ext: '.png'
     }
 
     await storage.uploadFile(fileContent)
+    expect(uploadSpy).toHaveBeenCalled()
+    expect(uploadSpy).toBeCalledWith(expect.objectContaining({
+      ...fileContent,
+      width: 824,
+      height: 44
```

Comment:
Why not use FileMetada type and include width and height few lines above? It's less code and cleaner to read I think